docs/migration-apiserver-to-crds.md

```diff
@@ -92,6 +98,27 @@ You can run the `service-catalog` binary with the `migration` parameter which tr
 | controller-manager-deployment | Provides the Controller Manager deployment name. |
 | apiserver-deployment | Provides the Apiserver deployment name. It is required only for the `backup` phase. |
 
+### Implementation details
+
+In order to get a consistent backup we have to make sure that no resource are modified during the backup process.
+To achieve that at the beginning of backup process migration tool creates a `ValidatingWebhookConfiguration` 
+to intercept and reject all attempts to mutate Service Catalog resources. Because of a limitation of a setup with Agregation API server (used by previous version of Service Catalog) this webhook call fails with a following message:
```

Comment:
```suggestion
to intercept and reject all attempts to mutate Service Catalog resources. Because of the setup limitation of the Aggregated API Server used in the previous version of Service Catalog, this webhook call fails with the following message:
```